bot/exts/info/pypi.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from random import choice
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
```

Comment:
I don't quite like losing the context of the `commands` name space here, but that's just my opinion.

Reply:
It must be done to be consistent with the rest of the bot.